### PR TITLE
fix `beatjump` description and beatsize range

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -510,9 +512,9 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
                    [PreviewDeckN],beatjump
                    [SamplerN],beatjump
 
-   Jump forward by :mixxx:coref:`beatjump_size <[ChannelN],beatjump_size>` (positive) or backward by :mixxx:coref:`beatjump_size <[ChannelN],beatjump_size>` (negative). If a loop is active, the loop is moved by :mixxx:coref:`beatjump_size <[ChannelN],beatjump_size>`.
+   Jump forward (positive) or backward (negative) by N beats. If a loop is active, the loop is moved by X beats.
```
Comment (Holzhaus) [inline source/chapters/appendix/mixxx_controls.rst:515]: Might make sense to split these long lines and put each sentence on a separate lines if you touch them anyway. Your decision.

**Review by Holzhaus — APPROVED** [review]
LGTM.

File: source/chapters/appendix/mixxx_controls.rst (hunk 1 above)
Comment (ronso0) [reply source/chapters/appendix/mixxx_controls.rst:515]: I'm not sure whether the line breaks would really be helpful here. If applied consistently for all controls, the page would grow enormously when being viewed on mobile screens.

Comment (Holzhaus) [reply source/chapters/appendix/mixxx_controls.rst:515]: Oh, I meant regular line break to make diff nicer to read. I thought this was not make da difference in the resulting html unless you put a blank line in between. Is that incorrect?

**Pull request merged**